site/_data/i18n/tags.yml

```diff
@@ -211,3 +211,12 @@ chrome70:
 
 insider:
   en: Chrome Dev Insider
+
+web-vitals:
```

Comment:
I copied this over from web.dev, but are we going to regret not using `core-web-vitals` from the start over here?

cc @philipwalton too

Reply:
INP isn't a core web vital, so this is accurate. This has _always_ been a weird point of contention, though.

Reply:
actually he's OoO at the moment. Since it'll just be the single use of the tag, and no one is likely to have a permalink to it any time soon, we'll call it ok for now and I can always open a PR to correct it if anyone feels strongly?

Reply:
> INP isn't a core web vital, so this is accurate

fair point. CWV-related? :)